admin/doc-requirements.txt

```diff
@@ -1,4 +1,5 @@
-Sphinx == 5.0.2
+# RTD theme does not work with >= 7 https://github.com/readthedocs/readthedocs.org/issues/10279
+Sphinx < 7
```

Comment:
Is this something we'll have to address at some point so we aren't stuck in the past?